setup.py

```diff
@@ -276,7 +276,8 @@ def _pkg_config(name):
                 subprocess.check_output(command_cflags).decode("utf8").strip(),
             )[::2][1:]
             return libs, cflags
-        except Exception:
+
+        except subprocess.CalledProcessError:
```

Comment:
The Windows build in our CI are failing with this change - https://github.com/python-pillow/Pillow/actions/runs/4614673587/jobs/8157938584

I'm guessing that this doesn't relate to your problem, but is just something that you thought you would do while you were here?

Reply:
Yes. This is something I used while looking into the issue.
As it's not a requirement for the change I've just dropped that commit.